llvm/test/MC/AMDGPU/gfx12_asm_sopp.s

```diff
@@ -69,15 +69,6 @@ s_wait_alu depctr_va_sdst(3)
 s_wait_alu depctr_va_vdst(14) depctr_va_sdst(6) depctr_vm_vsrc(6)
 // GFX12: encoding: [0x9b,0xed,0x88,0xbf]
 
-s_inst_prefetch 0x1234
```

Comment:
Please add one example of this instruction to `test/MC/AMDGPU/gfx12_unsupported.s`.

Reply:
Done